features/admin.applications.v1/components/forms/inbound-oidc-form.tsx

```diff
@@ -1272,6 +1420,24 @@ export const InboundOIDCForm: FunctionComponent<InboundOIDCFormPropsInterface> =
             };
         }
 
+        if(showHybridFlowEnableConfig) {
+            inboundConfigFormValues = {
+                ...inboundConfigFormValues,
+                hybridFlow: {
+                    enable: values.get("enable-hybrid-flow")?.length > 0,
+                    responseType: values.get(ApplicationManagementConstants.HYBRID_FLOW_RESPONSE_TYPE)
+                }
+            };
+        } else {
+            inboundConfigFormValues = {
+                ...inboundConfigFormValues,
+                hybridFlow: {
+                    enable: false,
+                    responseType: null
+                }
+            };
+        }
```

Comment:
isn't this code snippet repeated ?

Reply:
yes but it comes under two different functions. One is for submitting SPAs (updateConfigurationForSPA), and the other is for other applications (updateConfiguration)